src/Components/Dashboard.js

```diff
@@ -1,43 +1,82 @@
-import { useState } from 'react';
-
+import { useState, useEffect } from 'react';
+import React from 'react';
+import ContactCard from './ContactCard';
 
 export const Dashboard = ({ user, clients }) => {
     const [ highClientCard, setHigh ] = useState({});
     const [ lowClientCard, setLow ] = useState({});
-    const [ featured, setClientCard ] = useState({});
+    const [ featuredClient, setClientCard ] = useState({});
 
-    console.log('DASHBOARD USER', user)
-    console.log('DASHBOARD CLIENTS', clients.length)
-// if we want to try to find high/low client...
+    // console.log('DASHBOARD USER', user)
+    // console.log('DASHBOARD CLIENTS', clients)
 
-// const sortedClients = clients.sort((a, b) => a.averageRating - b.averageRating)
-// console.log(sortedClients)
-// const mapClients = clients.map(client => client.email )
-// console.log(mapClients)
-// const topClient = sortedClients[0]
-// setHigh(topClient);
 
 // otherwise, we could just do a featured client of the day...
-const featuredClient = clients[Math.floor(Math.random() * clients.length)];  
-console.log(featuredClient)
-setClientCard(featuredClient); 
+const randomClient = clients[Math.floor(Math.random() * clients.length)];  
+// console.log("gimmeRandom", randomClient.reviews[0].body)
+
+useEffect(() => {
+    if (randomClient) {
+        setClientCard(randomClient);
+        // console.log("feature", featuredClient.email)
+    }
+
+    if(clients) {
+        setHigh(sortByRatings(clients).pop())
+        setLow(sortByRatings(clients).shift())
+        console.log("highhhhhhh", highClientCard)
+        console.log("lowwwwwwww", lowClientCard)
+    }
+  }, [randomClient, clients]);
+
+
+const sortByRatings = (clients) => {
+    const allClients = [...clients]
+
+    const sortAverages = allClients.sort((a, b) => a.averageRating - b.averageRating)
+    return sortAverages
+  }
+
 
     return (
         <section>
-            {/* <h2>Welcome back, {userName}!</h2> */}
+            <h2>Welcome Back to ClienTell</h2>
```

Comment:
Did we want to put this back in eventually? like welcome back, {user.email} or do you think that will look weird?